src/components/ProductsPage/ProductsPage.tsx

```diff
@@ -1,34 +1,36 @@
-import { Divider, PageSection, PageSectionVariants, Text, TextContent } from '@patternfly/react-core';
-
 import { PageTitles } from 'common/constants';
 
+import { useQueryParamsEffect } from 'hooks/useQueryParamsEffect';
+import { useServiceContainer } from 'hooks/useServiceContainer';
 import { useTitle } from 'hooks/useTitle';
 
 import { ActionButton } from 'components/ActionButton/ActionButton';
 import { ContentBox } from 'components/ContentBox/ContentBox';
+import { PageLayout } from 'components/PageLayout/PageLayout';
+import { ProductsList } from 'components/ProductsList/ProductsList';
+
+import * as productApi from 'services/productApi';
+
+interface IProductsPageProps {
+  componentId?: string;
+}
+
+export const ProductsPage = ({ componentId = 'p1' }: IProductsPageProps) => {
+  const serviceContainerProducts = useServiceContainer(productApi.getProducts);
+
+  useQueryParamsEffect(serviceContainerProducts.run, { componentId });
 
-export const ProductsPage = () => {
   useTitle(PageTitles.products);
+
   return (
-    <>
-      <PageSection variant={PageSectionVariants.light}>
-        <TextContent>
-          <Text component="h1">ProductsPage</Text>
-          <Text component="p">
-            Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam quos unde, accusantium excepturi ad praesentium.
-          </Text>
-        </TextContent>
-      </PageSection>
-
-      <Divider component="div" />
-
-      <PageSection>
-        <ContentBox padding>
-          <ActionButton link="/products/100/versions/100/milestones/101/details">
-            Product Milestone (testing purposes)
-          </ActionButton>
-        </ContentBox>
-      </PageSection>
-    </>
+    <PageLayout title={PageTitles.products} description={<>This page contains products... TODO</>}>
```

Comment:
Description `This page contains list of products like <Label>EAP</Label>, usually product is a deliverable package composed of multiple projects like <Label>JBoss Modules</Label> or <Label>Hibernate<Label>.`

Reply:
@jbartece is there anything else you would like to propose for the page description that will be displayed on product list page?